### PR TITLE
Split up hadoop implementations [full build]

### PR DESCRIPTION
**Pull request by farodin91**
Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

Important part: 

- janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/MapReduceIndexManagement.java
- janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/StoreManager.java
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraHadoopStoreManager.java

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/MapReduceIndexManagement.java [hunk 1]
```diff
@@ -27,14 +26,8 @@
...
 import org.janusgraph.diskstorage.BackendException;
-import org.janusgraph.diskstorage.cassandra.AbstractCassandraStoreManager;
-import org.janusgraph.diskstorage.cassandra.astyanax.AstyanaxStoreManager;
-import org.janusgraph.diskstorage.cassandra.embedded.CassandraEmbeddedStoreManager;
-import org.janusgraph.diskstorage.cassandra.thrift.CassandraThriftStoreManager;
 import org.janusgraph.diskstorage.configuration.ConfigElement;
-import org.janusgraph.diskstorage.cql.CQLStoreManager;
-import org.janusgraph.diskstorage.hbase.HBaseStoreManager;
-import org.janusgraph.diskstorage.keycolumnvalue.KeyColumnValueStoreManager;
+import org.janusgraph.diskstorage.keycolumnvalue.StoreManager;
```
Comment (FlorianHockmann) [inline janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/MapReduceIndexManagement.java]: Codacy Issue found: [Avoid unused imports such as 'org.janusgraph.diskstorage.keycolumnvalue.StoreManager'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 10
           

Complexity increasing per file
==============================
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLHadoopStoreManager.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobs.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseHadoopStoreManager.java  1
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraHadoopStoreManager.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java  1
         

Clones added
============
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java  2
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835&bid=16195972)

File: janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class CassandraMapReduceIndexJobs {
+
+    private static final Logger log =
```
Comment (FlorianHockmann) [inline janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java]: Codacy Issue found: [Avoid unused private fields such as 'log'.](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java [hunk 3]
```diff
@@ -76,6 +76,7 @@
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 
+import org.apache.hadoop.mapreduce.InputFormat;
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java]: Codacy Issue found: [Avoid unused imports such as 'org.apache.hadoop.mapreduce.InputFormat'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobs.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class HBaseMapReduceIndexJobs {
+
+    private static final Logger log =
```
Comment (FlorianHockmann) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobs.java]: Codacy Issue found: [Avoid unused private fields such as 'log'.](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+import org.janusgraph.hadoop.scan.CQLHadoopScanRunner;
+import org.janusgraph.util.system.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class CQLMapReduceIndexJobs {
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java]: Codacy Issue found: [The utility class name 'CQLMapReduceIndexJobs' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java [hunk 6]
```diff
@@ -90,6 +91,9 @@
 import org.janusgraph.diskstorage.keycolumnvalue.StandardStoreFeatures;
 import org.janusgraph.diskstorage.keycolumnvalue.StoreFeatures;
 import org.janusgraph.diskstorage.keycolumnvalue.StoreTransaction;
+import org.janusgraph.hadoop.CQLHadoopStoreManager;
+import org.janusgraph.hadoop.HadoopStoreManager;
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java]: Codacy Issue found: [Avoid unused imports such as 'org.janusgraph.hadoop.HadoopStoreManager'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+import org.janusgraph.hadoop.scan.CassandraHadoopScanRunner;
+import org.janusgraph.util.system.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class CassandraMapReduceIndexJobs {
```
Comment (FlorianHockmann) [inline janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobs.java]: Codacy Issue found: [The utility class name 'CassandraMapReduceIndexJobs' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobs.java [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+import org.janusgraph.hadoop.scan.HBaseHadoopScanRunner;
+import org.janusgraph.util.system.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class HBaseMapReduceIndexJobs {
```
Comment (FlorianHockmann) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobs.java]: Codacy Issue found: [The utility class name 'HBaseMapReduceIndexJobs' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java [hunk 9]
```diff
@@ -90,6 +91,9 @@
...
+import org.janusgraph.hadoop.formats.cql.CqlBinaryInputFormat;
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java]: Codacy Issue found: [Avoid unused imports such as 'org.janusgraph.hadoop.formats.cql.CqlBinaryInputFormat'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

File: janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+    private static final Logger log =
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobs.java]: Codacy Issue found: [Avoid unused private fields such as 'log'.](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLHadoopStoreManager.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraHadoopStoreManager.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseHadoopStoreManager.java  1
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobsUtils.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobsUtils.java  1
         

Clones added
============
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  2
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobsUtils.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835&bid=16195972)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseHadoopStoreManager.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraHadoopStoreManager.java  1
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobsUtils.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobsUtils.java  1
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLHadoopStoreManager.java  1
         

Clones added
============
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CQLMapReduceIndexJobsUtils.java  2
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835&bid=16195972)

**Comment by farodin91** [comment]
@porunov I improved the changelog again a bit more and renamed CqlHadoopStoreManager and cqlMapReduceIndexJobsUtils.

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CqlMapReduceIndexJobsUtils.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseMapReduceIndexJobsUtils.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  1
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraHadoopStoreManager.java  1
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CqlHadoopStoreManager.java  1
- janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/hadoop/HBaseHadoopStoreManager.java  1
         

Clones added
============
- janusgraph-cql/src/main/java/org/janusgraph/hadoop/CqlMapReduceIndexJobsUtils.java  2
- janusgraph-cassandra/src/main/java/org/janusgraph/hadoop/CassandraMapReduceIndexJobsUtils.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4889835&bid=16195972)

**Comment by farodin91** [comment]
@porunov Do you want to check again? The changes?

**Comment by farodin91** [comment]
@porunov I wait for the build to finish.

**Pull request merged**